common/utils/src/test/scala/org/apache/spark/util/LogKeySuite.scala

```diff
@@ -25,9 +32,59 @@ class LogKeySuite
     extends AnyFunSuite // scalastyle:ignore funsuite
     with Logging {
 
-  test("LogKey enumeration fields must be sorted alphabetically") {
-    val keys = LogKey.values.toSeq
-    assert(keys === keys.sortBy(_.toString),
-      "LogKey enumeration fields must be sorted alphabetically")
+  // scalastyle:off line.size.limit
+  /* Used to regenerate the LogKey class file. Run:
+   {{{
+      SPARK_GENERATE_GOLDEN_FILES=1 build/sbt \
+        "common-utils/testOnly *LogKeySuite -- -t \"LogKey enumeration fields are correctly sorted\""
```

Comment:
Shall we remove the `-t ...` from the instruction to make it simpler?

Reply:
Sure